.github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md

```diff
@@ -3,6 +3,9 @@ name: Enviar software para revisión
 about: Usa esta plantilla para enviar un software a revisar en Español (Experimental)
 ---
 
+<!---
+En la sección que sigue a continuación, introduzca los dos valores de (1) el mango de GitHub del autor del paquete y (2) la URL del repositorio. También se pueden especificar valores para autores de paquetes adicionales. Los valores para cada uno deben colocarse entre las dos secciones HTML, como se ilustra para el valor "author1". Los símbolos HTML que definen cada variable deben dejarse como están - así que el autor que envía se especifica sustituyendo sólo "@github_handle", mientras que se dejan los símbolos HTML adjuntos como están. Este comentario puede ser eliminado una vez que haya sido leído y comprendido.
```

Comment:
LGTM except for one small thing: The literal translation of "handle" to "mango" makes no sense to me. Instead I suggest "usuario" (for user). Also I suggest replacing the masculine "del autor" with "la persona que envio el paquete" or the feminine "la autora".

```
... (1) el [-mango-]{+usuario+} de GitHub [-del autor del-]{+de la persona que envio el+} paquete
```

Same here

```
así que [-el autor-]{+la persona+} que envía se especifica sustituyendo sólo "@github_handle"
```

Reply:
Thanks @maurolepore, amended as suggested.